limit_to_employees/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Limit To Employees",
+
+    'summary': """
+        Limit To Employees""",
+
+    'description': """
+       When user do following action in odoo, it only shows internal users contacts in dropdown.
```

Comment:
"When users do the following actions..."

Reply:
There is description written in the next line point by point.

Reply:
Yes, I saw that. Just gave a hint how to fix the grammar there.